pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -31,8 +32,11 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/errors"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
 )
 
+const azureCredentialsFilePath = "/home/ubuntu/azure-credentials.yaml"
```

Comment:
could you provide a comment that explains what this is? if I were to run this roachtest on my laptop, would i need to create this?

Reply:
perhaps instead, we should put this stuff in some temp file path that gets deleted after a test run?

Reply:
oh is this getting put on each roachprod machine?

Reply:
Oh right, while running this I remembered that the problem with attempting to run this locally is that the credentials for the Azure app i created for this have permissions for the bucket, but don't have permission to create instances. So currently you can only run it via teamcity.

Reply:
> So currently you can only run it via teamcity.
sadness. So i assume you've test run this on TC? If so, i think this is good to merge!

Is there an open issue for dev inf to set up these managed identities? perhaps this is on testeng? i'm not sure.

Reply:
> Not sure if there's an open issue, but @golgeek mentioned that it is something he is working on.

Just created #149811